tests/runtime/engine/runner.py

```diff
@@ -321,11 +335,26 @@ def run_test(test):
                 print('\tCLEANUP error: %s' % e.stderr)
                 return Runner.FAIL
 
+        def print_befores_and_after_output():
+            if len(befores_output) > 0:
+                print("\tBefore cmd output:")
+                for (out, err) in befores_output:
+                    out = out.decode("utf-8")
+                    err = err.decode("utf-8")
+                    print(f"stdout: [{out}]\nstderr: [{err}]")
```

Comment:
Perhaps would be nicer with indentation? Also, I'm not sure what the purpose of the square brackets is, I'd remove them:
```suggestion
                    print(f"\t\tstdout: {out}\n\t\tstderr: {err}")
```
Same below for `AFTER`.